perllib/FixMyStreet/App/Controller/Reports.pm

```diff
@@ -378,6 +378,18 @@ sub load_and_group_problems : Private {
         non_public => 0,
         state      => [ keys %$states ]
     };
+    my $filter = {
+        order_by => $c->stash->{sort_order},
+        rows => $c->cobrand->reports_per_page,
+    };
+
+    if (defined $c->stash->{filter_status}{shortlisted}) {
+        $where->{'me.id'} = { '=', \"user_planned_reports.report_id"};
+        $where->{'user_planned_reports.removed'} = { 'IS', \"NULL" };
```

Comment:
` $where->{'user_planned_reports.removed'} = undef;` is a more idiomatic way of doing this.